### PR TITLE
Fixes #768 :Improved Code Quality

### LINKED ISSUE #768: Code Quality of header.html needs to be improved
### Expected results
Good code quality
### Actual results
Bad indentation with blank spaces
### Steps to reproduce the issue
Fix it
## Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by saarthakchats**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showing after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request
Improved indentation and removed blank spaces from header.html to improve code quality.

Fixes #768

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -20,8 +18,6 @@
         <link href="https://fonts.googleapis.com/css?family=Cabin+Sketch" rel="stylesheet">
         <link href='https://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600' rel='stylesheet' type='text/css'>
 		<link rel="icon" href="favicon.ico" />
```
Comment (nzec) [inline _includes/header.html]: Fix this.

File: _includes/header.html [hunk 2]
```diff
@@ -3,10 +3,8 @@
 	<head>
 		<meta charset="utf-8">
 		<title>FOSSASIA | Google Code-In 2017</title>
-
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-        <link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all" />
+		<link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all" />
         <link href="css/flexslider.css" rel="stylesheet" type="text/css" media="all" />
```
Comment (nzec) [inline _includes/header.html]: These as well.

File: _includes/header.html [hunk 3]
```diff
@@ -49,59 +45,56 @@
...
-						</ul>
-					</li>	
+										<li>
+										<a href="{{ baseurl }} ./#fossasia-projects">Open-Event Android App</a>
+										</li>
+										<li>
+										<a href="{{ baseurl }} ./#sus">Susi.ai</a>
+										</li>
+										<li>
+										<a href="{{ baseurl }} ./#loki">Loklak Search</a>
```
Comment (nzec) [inline _includes/header.html]: Fix these.

File: _includes/header.html [hunk 4]
```diff
@@ -49,59 +45,56 @@
...
-		                                    <a href="https://fossasia.org/">
-		                                        Main Site
-		                                    </a>
+		                                    <a href="https://fossasia.org/">Main Site</a>
+		                                </li>
+		                                <li>
+		                                    <a href="http://blog.fossasia.org/">Blog</a>
 		                                </li>
 		                                <li>
 											<a href="#">Events</a>
```
Comment (nzec) [inline _includes/header.html]: These.

**Comment by nzec** [comment]
@saarthakchats There are a lot of problems. Please fix them.

**Comment by saarthakchats** [comment]
@duttaditya18 Fixed

File: _includes/header.html [hunk 5]
```diff
@@ -49,73 +45,70 @@
...
-						</li>
-						</ul>
-					</li>	
+										<li>
+										<a href="{{ baseurl }} ./#fossasia-projects">Open-Event Android App</a>
+										</li>
+										<li>
+										<a href="{{ baseurl }} ./#sus">Susi.ai</a>
+										</li>
+										<li>
```
Comment (nzec) [inline _includes/header.html]: These still need fixing.

**Comment by nzec** [comment]
Fix this as well.

Problem

**Comment by saarthakchats** [comment]
@duttaditya18 
Is there anything else?

**Comment by nzec** [comment]
@saarthakchats These is. If you are fixing indentation, pay close attention.

Problem
---
Problem
---
Problem
---

**Comment by saarthakchats** [comment]
@codedsun Pls review

**Comment by nzec** [comment]
@saarthakchats [Line 126](https://github.com/saarthakchats/tae/blob/09cc2977257abf4b7d289db83f26a7fdc2e1978b/_includes/header.html#L126) and [Line 36](https://github.com/saarthakchats/tae/blob/09cc2977257abf4b7d289db83f26a7fdc2e1978b/_includes/header.html#L36) were still not fixed.

**Review by nzec — APPROVED** [review]
Finally! Good job!

**Comment by saarthakchats** [comment]
@duttaditya18 
Alas, thanks

File: _includes/header.html [hunk 6]
```diff
@@ -3,25 +3,21 @@
...
-        <link href="css/flexslider.css" rel="stylesheet" type="text/css" media="all" />
-        <link href="css/stylesheet.css" rel="stylesheet" type="text/css" media="all" />
-        <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all" />
-        <link href="css/theme-red.css" rel="stylesheet" type="text/css" media="all" />
-        <link href="css/animate.css" rel="stylesheet" type="text/css" media="all" />
-        <link href="css/hover.css" rel="stylesheet" type="text/css" media="all" />
+		<link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all" />
+        	<link href="css/flexslider.css" rel="stylesheet" type="text/css" media="all" />
+        	<link href="css/stylesheet.css" rel="stylesheet" type="text/css" media="all" />
+       		<link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all" />
```
Comment (ParthS007) [inline _includes/header.html]: Indent it properly.

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Comment by saarthakchats** [comment]
@ParthS007 Which line are you talking about, it looks fine

**Comment by saarthakchats** [comment]
@ParthS007 
Which line?

**Comment by harshit98** [comment]
> @ParthS007
Which line?

@saarthakchats Line no. 10.

**Comment by saarthakchats** [comment]
@harshit98 
It looks fine. I guess...


On the commits' section, it looks like this
screenshot 18

But when I click edit, it is fine...
screenshot 19

**Comment by saarthakchats** [comment]
@vaibhavsingh97 
I think it should be fine now

**Review by vaibhavsingh97 — APPROVED** [review]

**Review by the-codingninja — APPROVED** [review]

**Comment by abishekvashok** [comment]
@saarthakchats don't use the online editor, it is lame and dump. Use a local editor.

**Pull request merged**